extensions/ql-vscode/src/run-queries.ts

```diff
@@ -353,26 +353,6 @@ export class QueryEvaluationInfo {
       });
   }
 
-  /**
-   * Calls the appropriate CLI command to generate a JSON log summary and parse it 
-   * into the appropriate data model for the log visualizer. 
-   */
-  parseJsonLogSummary(cliServer: cli.CodeQLCliServer): void {
-    void cliServer.generateJsonLogSummary(this.evalLogPath, this.jsonEvalLogSummaryPath)
-      .then(() => {         
-        // TODO(angelapwen): Stream the file in. 
-        fs.readFile(this.jsonEvalLogSummaryPath, (err, buffer) => {
-          if (err) {
-            throw new Error(`Could not read structured evaluator log summary JSON file at ${this.jsonEvalLogSummaryPath}.`);
-          }
-          parseVisualizerData(buffer.toString()); // Eventually this return value will feed into the tree visualizer.
-        });
-      })
-      .catch(err => {
-        void showAndLogWarningMessage(`Failed to generate JSON structured evaluator log summary. Reason: ${err.message}`);
```

Comment:
You lose this error handler, but I think that's covered by the CLI server's default error handling.

Reply:
I thought this would be ok given the user doesn't really know that there is a JSON structured log summary that the viewer is going to consume (it may unnecessarily consume them given that they can still view the raw JSON and summary text logs).